app/src/main/java/org/wikipedia/feed/suggestededits/SuggestedEditsFeedClient.kt

```diff
@@ -32,6 +33,7 @@ class SuggestedEditsFeedClient(private var action: DescriptionEditActivity.Actio
     private var targetSummary: SuggestedEditsSummary? = null
     private val langFromCode: String = app.language().appLanguageCode
     private val langToCode: String = if (app.language().appLanguageCodes.size == 1) "" else app.language().appLanguageCodes[1]
+    private var page: MwQueryPage? = null
```

Comment:
Does `page` need to be a member variable here? Couldn't it just be a parameter of `toSuggestedEditsCard()`?

Reply:
Done